configs/ttalps_skimmer_looseSemileptonic_config.py

```diff
@@ -67,6 +67,8 @@
 treeOutputFilePath = f"{basePath}/{sampleName}/skimmed_looseSemileptonic/{fileName}"
 
 weightsBranchName = "genWeight"
+eventsTreeNames = ("Events",)
+muonMatchingType = ""
```

Comment:
I guess here we will have different options for matching - it's best to avoid strings where possible, but at least you should list possible options in a comment above this variable (otherwise I have no way of knowing what I can put there unless I check in the C++ class).

Unless... I see in other files that it's more of an empty variable that will never be used. Can we implement the reading of this variable in the C++ class such that it is allowed not to be present in the config at all? I think I've done it like this in a few places, with try-catch blocks.